tests/manage/storageclass/test_create_multiple_sc_with_same_pool_name.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+import logging
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import defaults
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import (
+    create_ceph_block_pool,
+    create_rbd_secret, create_cephfs_secret,
+    create_project
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def resources(request):
+    """
+       Delete the resources created during the test
+       Returns:
+           tuple: empty lists of resources
+       """
+    pods, pvcs, storageclasses = ([] for i in range(3))
+
+    def finalizer():
+        """
+            Delete the resources created during the test
+            """
+        for resource_type in pods, pvcs, storageclasses:
+            for resource in resource_type:
+                resource.delete()
+                resource.ocp.wait_for_delete(resource.name)
+
+    request.addfinalizer(finalizer)
+
+    return pods, pvcs, storageclasses
+
+
+@tier1
+@pytest.mark.usefixtures(
+    create_project.__name__,
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+)
+class TestCreateMultipleScWithSamePoolName(ManageTest):
+    """
+    Create Multiple Storage Class with same pool name
+    """
+    @pytest.mark.parametrize(
+        argnames="interface_type",
+        argvalues=[
+            pytest.param(
+                *[
+                    constants.CEPHBLOCKPOOL
+                ], marks=pytest.mark.polarion_id("OCS-622")
+            ),
+            pytest.param(
+                *[
+                    constants.CEPHFILESYSTEM
+                ], marks=pytest.mark.polarion_id("OCS-623")
+            )
+        ]
+    )
+    def test_create_multiple_sc_with_same_pool_name(
+            self, interface_type, resources
+    ):
+        """
+        This test function does below,
+        *. Creates multiple Storage Classes with same pool name
+        *. Creates PVCs using each Storage Class
+        *. Mount each PVC to an app pod
+        *. Run IO on each app pod
+        """
+        # Unpack resources
+        pods, pvcs, storageclasses = resources[:3]
```

Comment:
why not just pods, pvcs, storageclasses = resources ?

Reply:
Done